chronicle/chronicleApp.swift

```diff
@@ -9,9 +9,19 @@ import SwiftUI
 
 @main
 struct chronicleApp: App {
```

Comment:
Why is `chronicleApp` (lowercase `c`) not `ChronicleApp`?

Reply:
renamed